Dockerfile

```diff
@@ -0,0 +1,27 @@
+ARG NODE_VERSION=14.x
+
+FROM gcr.io/connectedcars-staging/node-builder.master:$NODE_VERSION as builder
+
+ARG SSH_KEY_PASSWORD
+ARG NPM_TOKEN
+ARG COMMIT_SHA=master
+
+WORKDIR /app
+
+RUN apt-get update
+
+USER builder
+
+# Copy application code.
+COPY --chown=builder:builder . /app
+
+RUN npm i
+
+# Run ci checks
+RUN npm run ci-audit
+
+RUN npm run ci-mocha
+
+RUN npm run ci-eslint
+
+RUN npm test
```

Comment:
Any reason for running both `ci-mocha` and `test`?

Reply:
Its so that the build fails if the tests fails. Like in https://github.com/connectedcars/node-setup/blob/master/Dockerfile

Reply:
I will add that comment message

Reply:
You should be able to add `--hard-fail` to ci-mocha, introduced to checks here: https://github.com/connectedcars/node-test/pull/31